setup.py

```diff
@@ -69,7 +69,7 @@
     include_package_data=True,
     python_requires=">=3.6",
     extras_require={
-        'cplex': ["cplex; python_version >= '3.6' and python_version < '3.8'"],
+        'cplex': ["cplex; python_version >= '3.6' and python_version < '3.9'"],
```

Comment:
A similar change was done when supporting 3.9 in Aqua. I will note that since we only support Python 3.6 or above, as per the python requires, the python_version >= 3.6 is really not needed here (in Aqua it just has < 3.9 for this entry